openapi_to_fastapi/tests/data/ihan/CompanyBasicInfo.json

```diff
@@ -42,7 +42,25 @@
               }
             }
           }
-        }
+        },
+        "parameters": [
```

Comment:
Is there any special way to mark these as optional?

Reply:
if there's no `"required": true` set, then they're considered optional